Index: Melody1Project/templates/Movies/cart.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Melody1Project/templates/Movies/cart.html b/Melody1Project/Movies/templates/Movies/cart.html
rename from Melody1Project/templates/Movies/cart.html
rename to Melody1Project/Movies/templates/Movies/cart.html
--- a/Melody1Project/templates/Movies/cart.html	(revision acd735e825d21297a3bfadd13c2b618660592302)
+++ b/Melody1Project/Movies/templates/Movies/cart.html	(date 1738536952340)
@@ -21,7 +21,7 @@
                 </tr>
             </thead>
             <tbody>
-            {%for movie in template_data.movies_in_cart%}
+            {% for movie in template_data.movies_in_cart %}
                 <tr>
                     <td>{{movie.id}}</td>
                     <td>{{movie.title}}</td>
diff --git a/Melody1Project/templates/images/film.svg b/Melody1Project/Movies/templates/images/film.svg
rename from Melody1Project/templates/images/film.svg
rename to Melody1Project/Movies/templates/images/film.svg
diff --git a/Melody1Project/templates/Movies/index.html b/Melody1Project/Movies/templates/Movies/index.html
rename from Melody1Project/templates/Movies/index.html
rename to Melody1Project/Movies/templates/Movies/index.html
diff --git a/Melody1Project/templates/Movies/base.html b/Melody1Project/Movies/templates/Movies/base.html
rename from Melody1Project/templates/Movies/base.html
rename to Melody1Project/Movies/templates/Movies/base.html
